packages/edit-site/src/components/global-styles/font-sizes/font-size.js

```diff
@@ -53,10 +53,10 @@ function FontSize() {
 
 	// Navigate to the font sizes list if the font size is not available.
 	useEffect( () => {
-		if ( ! fontSize ) {
-			goTo( '/typography/font-sizes/', { isBack: true } );
```

Comment:
Using `goTo` here was hiding a bug, where `goTo` was being called twice. The fix is to also check for the `slug`, which then makes the code only run once